backend/pom.xml

```diff
@@ -1,9 +1,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.jboss</groupId>
-    <artifactId>dependency-analyzer</artifactId>
+    <groupId>org.jboss.da</groupId>
```

Comment:
It is right, that it is in the PNC space right now, but it is held as a separate project. Hmm. Currently we were speaking about it to held it completely separated - only REST API will be connection point. For now my vote is to keep the groupId separated. 
@matejonnet @rnc Wdyt?